tests/test_distro.py

```diff
@@ -1377,8 +1376,29 @@ def test_openelec6_release(self):
             'minor_version': '0',
         }
         self._test_outcome(desired_outcome)
-        
-        
+
+    def test_mandriva2011_release(self):
+        desired_outcome = {
+            'id': 'mandrivalinux',
```

Comment:
It's so bad that different files return different information. We really need that framework.. meh :(